test/plugin/test_out_google_cloud.rb

```diff
@@ -195,6 +260,16 @@ def test_gce_metadata_does_not_load_when_use_metadata_service_is_false
     assert_equal false, d.instance.running_on_managed_vm
   end
 
+  def test_metadata_partial_overrides_on_gce
+    setup_gce_metadata_stubs
+    d = create_driver(CONFIG_MISSING_METADATA_PROJECT_ID)
+    d.run
+    assert_equal PROJECT_ID, d.instance.project_id
```

Comment:
Same here. Otherwise it's hard to tell what's the actual difference between those two tests.